services/web/server/src/simcore_service_webserver/users/api.py

```diff
@@ -318,6 +318,7 @@ async def update_expired_users(engine: Engine) -> list[UserID]:
         return await _db.do_update_expired_users(conn)
 
 
+# mypy: disable-error-code=truthy-function
```

Comment:
sam comment as above I would have put it at the top of the module, since the scope is the entire module.